Communitter/src/main/java/com/communitter/api/service/PostService.java

```diff
@@ -61,16 +61,50 @@ public Post createPost(Long id, Post post) {
     public void deletePost(Long communityId, Long id){
         Community targetCommunity = communityRepository.findById(communityId).orElseThrow(()->new NoSuchElementException("Community does not exist"));
```

Comment:
targetCommunity is never user after defined. it can be deleted.

Reply:
@ynsmrkrkmz its for checking if community exists or not, there is no further functionality for target community.